funfact/lang/_terminal.py

```diff
@@ -165,9 +166,11 @@ def __init__(self, *size, symbol=None, initializer=None, optimizable=True):
         self.symbol = self.TensorSymbol(symbol or self.uuid)
         self.initializer = initializer
         self.optimizable = optimizable
+        self._penalty = penalty
```

Comment:
I think we can leave it as a public member, just like `initializer` and `optimizable`.